### PR TITLE
Package by default for experimental line

### PR DESCRIPTION
**Pull request by olblak**
* By default package for experimental line
* Increase memory limit for windows pods

### CODE REVIEW

File: PodTemplates.d/package-windows [hunk 1]
```diff
@@ -22,11 +22,11 @@ spec:
...
     resources:
       limits:
-        memory: "1Gi"
-        cpu: "1"
+        memory: "8Gi"
+        cpu: "2"
       requests:
-        memory: "1Gi"
-        cpu: "1"
+        memory: "8Gi"
```
Comment (timja) [inline PodTemplates.d/package-windows]: does it really need to request 8gi? that's a lot ? will need quite big vms for that

**Comment by olblak** [comment]
In the association with this [commit](https://github.com/jenkinsci/packaging/commit/c25d3853f9dd46a90730f73b69eeb413af90a0be), not yet merged on jenkinsci/packaging atm, it allows us to push artifacts generated from release.ci.jenkins.io to a different folder than the official one, as an example here https://release.pkg.jenkins.io/debian-experimental/

**Comment by olblak** [comment]
versus https://pkg.jenkins.io/debian or https://pkg.jenkins.io/debian-stable

File: PodTemplates.d/package-windows (hunk 1 above)
Comment (olblak) [reply PodTemplates.d/package-windows]: There is already a windows node deployed as defined [here](https://github.com/jenkins-infra/azure/blob/master/plans/publick8s.tf#L128L142), last time I checked it wasn't possible to define a node pool with default value set to 0. In the futur we can reduce this limit according the need

**Pull request merged**